openmc/mgxs/library.py

```diff
@@ -382,9 +382,7 @@ def get_mgxs(self, domain, mgxs_type):
         ----------
         domain : Material or Cell or Universe or Integral
             The material, cell, or universe object of interest (or its ID)
-        mgxs_type : {'total', 'transport', 'absorption', 'capture', 'fission',
-                     'nu-fission', 'scatter', 'nu-scatter', 'scatter matrix',
-                     'nu-scatter matrix', 'chi'}
+        mgxs_type : {'total', 'transport', 'absorption', 'capture', 'fission', 'nu-fission', 'scatter', 'nu-scatter', 'scatter matrix', 'nu-scatter matrix', 'chi'}
```

Comment:
In general we keep to 80 characters, but in this instance (description of parameters, properties, etc), if you break up the line over multiple, it doesn't render correctly in the documentation and gives you a warning when running `make html`.